### PR TITLE
Release 1.0 (2016 Season)

### PR DESCRIPTION
**Pull request by ehsanmasdar**
Update for **2016-2017 Season**:
- Updated to newest FIRST library release
- Requires Android Studio 2.0 or greater

### CODE REVIEW

**Comment by ehsanmasdar** [comment]
@smo-key We should do the thing, you approve?

**Comment by smo-key** [comment]
@ehsanmasdar Can you write a line-item description for this? Seems like there are a whole lot more commits than just those to update the library version.

**Comment by ehsanmasdar** [comment]
@smo-key That was all done last season and never made it into master. I can't tackle a changelog right now but I'll look into it later.

File: build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```
Comment (smo-key) [inline build.gradle:7]: Note: May require Android Studio 2.0+.

File: ftc-library/src/main/java/com/lasarobotics/library/sensor/legacy/hitechnic/Gyroscope.java [hunk 2]
```diff
@@ -73,7 +73,7 @@ public void reset() {
      * @return The offset gyroscope rotation in degrees per second
      */
     public double getRate() {
-        return gyroSensor.getRotation() - offset;
+        return gyroSensor.getHeading() - offset;
```
Comment (smo-key) [inline ftc-library/src/main/java/com/lasarobotics/library/sensor/legacy/hitechnic/Gyroscope.java:76]: Slightly changes behavior of `getRate()` (probably for the better).

**Review by smo-key — APPROVED** [review]
@ehsanmasdar Looks good! :+1:

No breaking changes.

**Comment by smo-key** [comment]
@ehsanmasdar Do the honors. :)

**Comment by ehsanmasdar** [comment]
@homu r+

**Comment by ehsanmasdar** [comment]
@homu :(

**Comment by smo-key** [comment]
RIP @homu

**Comment by smo-key** [comment]
@homu r+

**Comment by smo-key** [comment]
@ehsanmasdar Check to make sure you have privileges to Homu.

**Comment by smo-key** [comment]
@ehsanmasdar I think @homu is broken.

**Pull request merged**